src/Protocol/Parser/Strategy/Resp3Strategy.php

```diff
@@ -95,6 +100,7 @@ protected function parseVerbatimString(string $string): array
         return [
             'type' => 'verbatimString',
```

Comment:
Pull the return type strings to a const that can be refered to

Reply:
Done!